<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_region</class>
 <widget class="QMainWindow" name="MainWindow_region">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账号注册</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>658</width>
      <height>364</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/new/prefix1/region2.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>51</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>年龄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>就诊医生</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>体温</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item>
      <widget class="QLineEdit" name="line_pass"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rbtn_man">
         <property name="text">
          <string>男</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbtn_woman">
         <property name="text">
          <string>女</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="line_age"/>
     </item>
     <item>
      <widget class="QLineEdit" name="line_doctname"/>
     </item>
     <item>
      <widget class="QLineEdit" name="line_tempr"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>281</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/new/prefix1/region_now.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_puls">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>脉搏</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_record">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>255</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>就诊记录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
